.changeset/grumpy-walls-nail.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+chore: rename `open-in-brower.ts` to `open-in-browser.ts`
```

Comment:
I think we will find that the changelogs get too crowded if we include changesets for things that have no user-facing implication.

Reply:
Is that such a bad thing? Changelogs shouldn't double up as announcement posts, which we should do separately.

Reply:
But in this case... yeah it's overkill. I did consider not doing it, but it would've broken my own rule.